### PR TITLE
Feature/tf https

### PR DESCRIPTION
**Pull request by steves-bjss**
Update cookie policy and rewrites

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -77,7 +77,7 @@ public static void ConfigureCookies(this IServiceCollection services, IConfigura
                 options.LogoutPath = "/Identity/Account/Logout";
 
                 // TODO: - This will need addressing. Its causing issues due to SSL termination in Azure
-                options.Cookie.SecurePolicy = CookieSecurePolicy.None;
+                options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
                 options.ExpireTimeSpan = cookieExpiration.ExpireTimeSpan;
```
Comment (MichaelRobinson-BJSS) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/ServiceCollectionExtensions.cs:81]: You can remove the TODO comments with this change

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/ServiceCollectionExtensions.cs [hunk 2]
```diff
@@ -86,7 +86,7 @@ public static void ConfigureCookies(this IServiceCollection services, IConfigura
             services.AddAntiforgery(options =>
             {
                 // TODO: - This will need addressing. Its causing issues due to SSL termination in Azure
-                options.Cookie.SecurePolicy = CookieSecurePolicy.None;
+                options.Cookie.SecurePolicy = CookieSecurePolicy.Always;
```
Comment (MichaelRobinson-BJSS) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/ServiceCollectionExtensions.cs:89]: And this TODO

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/ServiceCollectionExtensions.cs (hunk 1 above)
Comment (steves-bjss) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/ServiceCollectionExtensions.cs:80]: Updated

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/ServiceCollectionExtensions.cs (hunk 2 above)
Comment (steves-bjss) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/ServiceCollectionExtensions.cs:87]: Updated

**Review by MichaelRobinson-BJSS — DISMISSED** [review]

**Review by MichaelRobinson-BJSS — APPROVED** [review]

**Pull request merged**